build.sbt

```diff
@@ -95,6 +98,12 @@ lazy val shared = (project in file("shared"))
   .settings(commonSettings: _*)
   .settings(
     version := "0.1",
+    scalacOptions ++= Seq(
```

Comment:
Since you have this five times, might be worth creating a `val` for that.

Reply:
I honestly want to at the end of the week push that to commonSettings - when all modules have it 
but for now, maybe val makes sense...